implementation/configuration/src/configuration_impl.cpp

```diff
@@ -3255,16 +3255,17 @@ void configuration_impl::trim(std::string &_s) {
         std::find_if(
             _s.begin(),
             _s.end(),
-            [](unsigned char ch) { return !std::isspace(ch); }
+            [](auto const ch) { return !std::isspace(ch); }
```

Comment:
Prefer `const auto` over `auto const`.

```suggestion
            [](const auto ch) { return !std::isspace(ch); }
```